src/TurbulenceClosures/turbulence_closure_implementations/scalar_diffusivity.jl

```diff
@@ -0,0 +1,55 @@
+import Oceananigans.Grids: required_halo_size
+
+struct ScalarDiffusivity{TD, Dir, N, K} <: AbstractScalarDiffusivity{TD, Dir}
+    ν :: N
+    κ :: K
+
+    function ScalarDiffusivity{TD, Dir}(ν::N, κ::K) where {TD, Dir, N, K}
+        return new{TD, Dir, N, K}(ν, κ)
+    end
+end
+
+"""
+    ScalarDiffusivity([FT=Float64;]
+                         ν=0, κ=0, time_discretization = ExplicitTimeDiscretization())
+
+Returns parameters for an isotropic diffusivity model with viscosity `ν`
+and thermal diffusivities `κ` for each tracer field in `tracers`
+`ν` and the fields of `κ` may be constants, arrays, fields, or
+functions of `(x, y, z, t)`.
+
+`κ` may be a `NamedTuple` with fields corresponding to each tracer, or a
+single number to be a applied to all tracers.
+"""
+function ScalarDiffusivity(FT=Float64;
+                              ν=0, κ=0, direction=:ThreeDimensional, time_discretization::TD = ExplicitTimeDiscretization()) where {TD, Dir}
```

Comment:
```suggestion
                              ν=0, κ=0, direction=ThreeDimensional(), time_discretization::TD = ExplicitTimeDiscretization()) where {TD, Dir}
```

also given this "abbreviated" syntax, we may want to change `ExplicitTimeDiscretization` to just `Explicit`.

Reply:
(which we can do later)

Reply:
also should we call this `direction` or `isotropy`?

Reply:
(or something else...)

Reply:
I think isotropy is a good name, then the input to the constructor will be `isotropy::Iso=Threedimensional()`